.github/workflows/check.yml

```diff
@@ -0,0 +1,19 @@
+name: Check
+
+on: push
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up JDK 1.8
+      uses: actions/setup-java@v1
+      with:
+        java-version: 1.8
+    - name: Grant execute permission for gradlew
+      run: chmod +x gradlew
+    - name: Build with Gradle
```

Comment:
❓ Do we need to cache Gradle? Should we use https://github.com/marketplace/actions/gradle-command ?

Reply:
Replaced with it.